modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb

```diff
@@ -78,6 +78,18 @@
     end
   end
 
+  context 'pdf content verification' do
+    it 'generates the expected pdf' do
+      Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
+      path = described_class.new.generate_pdf(higher_level_review.id)
```

Comment:
does this pdf need to be cleaned up post-test?

Reply:
its writing to tmp, but we could destroy it to be safe

Reply:
you're call